javascript/javascript1/week2/Weatherwear.js

```diff
@@ -1,10 +1,9 @@
-const WeatherDressCode = temperature(18); 
-function WeatherDressCode (tempearature){
+ function WeatherDressCode(tempearature){
     if (tempearature < 18) {
         console.log("wear jacket");
```

Comment:
Again, you are not supposed to print these log messages as you did. Instead of printing strings, you must use a function to determine what to be printed.